scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1194,10 +1194,11 @@ def config_logger(self):
     fh.setLevel(logging.DEBUG)
     # create console handler to log messages ERROR and above
     ch = logging.StreamHandler()
-    ch.setLevel(logging.ERROR)
+    ch.setLevel(logging.DEBUG)
```

Comment:
The above comment states to log messages ERROR and above, but log level is set to DEBUG.
Any specific reason, it is required to set to DEBUG.